### PR TITLE
Changing "plugin" to "plug-in"

### PR DESCRIPTION
**Pull request by apinnick**
Adding hyphen to align with IBM Style Guide

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* We do not accept PRs for Foreman older than 3.3.

### CODE REVIEW

File: guides/common/modules/proc_installing-shellhooks-plugin.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-shellhooks-plugin.adoc:1]:
```suggestion
[id="installing-shellhooks-plug-in_{context}"]
```

File: guides/common/modules/proc_installing-webhooks-plugin.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-webhooks-plugin.adoc:2]:
```suggestion
[id="installing-webhooks-plug-in_{context}"]
= Installing Webhooks plug-in
```

It's a Foreman plug-in according to the installer command.

File: guides/common/modules/proc_uninstalling-the-foreman-templates-plugin.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_uninstalling-the-foreman-templates-plugin.adoc:2]:
```suggestion
[id="Uninstalling_the_Foreman_Templates_Plug-in_{context}"]
= Uninstalling the Foreman Templates plug-in
```

File: guides/common/modules/proc_creating-a-shellhook-to-print-arguments.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_creating-a-shellhook-to-print-arguments.adoc:10]:
```suggestion
** xref:installing-webhooks-plug-in_{context}[]
```

**Review by maximiliankolb — CHANGES_REQUESTED** [review]

File: guides/common/modules/proc_installing-webhooks-plugin.adoc
Comment (apinnick) [reply guides/common/modules/proc_installing-webhooks-plugin.adoc:2]: I agree. This didn't look right to me. Will fix.

File: guides/common/modules/proc_installing-webhooks-plugin.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-webhooks-plugin.adoc:2]:
```suggestion
= Installing the Webhooks plug-in
```

File: guides/common/modules/proc_installing-webhooks-plugin.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-webhooks-plugin.adoc:1]: please adjust the name of the file.

Comment (apinnick) [reply guides/common/modules/proc_installing-webhooks-plugin.adoc:1]: What should I name it? `proc_installing-the-webhooks-plug-in.adoc`? If I do that, I would probably have to fix all the other "installing ... plugin" files.

File: guides/common/modules/proc_installing-webhooks-plugin.adoc
Comment (apinnick) [reply guides/common/modules/proc_installing-webhooks-plugin.adoc:2]: fixed

File: guides/common/modules/proc_installing-webhooks-plugin.adoc
Comment (maximiliankolb) [reply guides/common/modules/proc_installing-webhooks-plugin.adoc:1]: Yes, that would be nice actually. But I am also OK with doing this in another PR. Your call, just lmk.

**Review by maximiliankolb — COMMENTED** [review]
One open thread that's not blocking this PR; rest LGTM!

File: guides/common/modules/proc_installing-webhooks-plugin.adoc
Comment (apinnick) [reply guides/common/modules/proc_installing-webhooks-plugin.adoc:1]: OK. When I have a lot of time on my hands, I will create a separate PR to fix file names. :-)

**Review by maximiliankolb — APPROVED** [review]

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked to 3.9: :heavy_check_mark: 
   03531d9964..f0e4d32fdf  3.9 -> 3.9

**Pull request merged**
